api4/user_test.go

```diff
@@ -259,7 +259,7 @@ func TestCreateUserWithInviteId(t *testing.T) {
 
 		_, resp := Client.CreateUserWithInviteId(&user, inviteId)
 		CheckNotFoundStatus(t, resp)
-		CheckErrorMessage(t, resp, "store.sql_team.get_by_invite_id.find.app_error")
+		CheckErrorMessage(t, resp, "store.sql_team.get_by_invite_id.finding.app_error")
```

Comment:
you need to update this keys in the en.json file

Reply:
Yes, the point here is I added early termination to some store methods, the behavior have changed slightly. Basically, if an error happens it sets the c.Err and directly returns, instead of fill the c.Err field and keep going.

Because that, the tests have a different error.